magento/magento2
# customer Orders query shipment comments created_at is returned in +0 timezone not in store configured timezone

### Description (*)
This pull request (PR) will be change created_at date to local timezone format date in customer order query
   

### Related Pull Requests
 I added function to change created_at date to local timezone format date.

### Fixed Issues (if relevant)

1. Fixes #36947 

### Manual testing scenarios (*)
1. Create order
2. Call customer Orders query and check created_at, order_date  field from orders
3. Result is that the time from both admin order view page and query be the same and in our configured timezone

<img width="1399" alt="Screenshot 2023-06-20 at 5 14 57 PM" src="https://github.com/magento/magento2/assets/88073031/e531f40b-3755-4c83-bb1b-6ec94b2bd96d">

<img width="494" alt="Screenshot 2023-06-20 at 5 14 41 PM" src="https://github.com/magento/magento2/assets/88073031/e2177be3-aacf-4fee-8d21-e2d366de20f8">





### Contribution checklist (*)
